java/vector/src/test/java/org/apache/arrow/vector/TestListVector.java

```diff
@@ -513,9 +513,9 @@ public void testNestedListVector() throws Exception {
       ArrayList<ArrayList<Long>> resultSet = (ArrayList<ArrayList<Long>>) result;
       ArrayList<Long> list;
 
-      assertEquals(2, resultSet.size());              /* 2 inner lists at index 0 */
-      assertEquals(3, resultSet.get(0).size());       /* size of first inner list */
-      assertEquals(4, resultSet.get(1).size());       /* size of second inner list */
+      assertEquals(2, resultSet.size()); /* 2 inner lists at index 0 */
+      assertEquals(3, resultSet.get(0).size()); /* size of first inner list */
```

Comment:
this seems to break formatting intended formatting.  I think I'm okay with it but is this what the "validateComments" option does in checkstyle.xml?

Reply:
@emkornfield Thanks for your comments. 
You are right. This is caused by the "validateComments" option. If we set it to false, the comment formatting can be preserved.